bot/cogs/coding/coding.py

```diff
@@ -23,11 +23,13 @@ class Coding(Cog):
     """To test code and check docs."""
 
     def __init__(self, bot: Bot) -> None:
+        """Initialize the coding."""
```

Comment:
Well, I always get a message about `__init__` not having a docstring. That's why

Reply:
As pointed out in another PR of yours, you can add that error/warning to your ignored errors for this repository since we don't enforce this and these comments are really unnecessary and only take up space

Reply:
In fact, the thing isn't that it's useless to put a docstring in `__init__` : it allows you to have a help on how to use the class constructor when you are writing it. It's just that I'm too lazy to correctly format it, but it could be good to put docstring in `__init__` for the public classes

Reply:
We could put a docstring in some of the `__init__` functions, I agree, but it needs to be a meaningful docstring, just putting `"Initialize the {class name}"` is unacceptable

Reply:
Anyway, I removed every single occurence of it. I think